tests/tensor/test_extra_ops.py

```diff
@@ -647,18 +634,38 @@ def test_grad(self, x_ndim, repeats_ndim, axis):
             repeats_size = (x_test.shape[axis] if axis is not None else x_test.size,)
         repeats = rng.integers(1, 6, size=repeats_size)
         utt.verify_grad(
-            lambda x: Repeat(axis=axis)(x, repeats),
+            lambda x: repeat(x, repeats, axis=axis),
             [x_test],
         )
 
-    def test_broadcastable(self):
-        x = TensorType(config.floatX, shape=(None, 1, None))()
-        r = Repeat(axis=1)(x, 2)
-        assert r.broadcastable == (False, False, False)
-        r = Repeat(axis=1)(x, 1)
-        assert r.broadcastable == (False, True, False)
-        r = Repeat(axis=0)(x, 2)
-        assert r.broadcastable == (False, True, False)
+    def test_static_shape(self):
+        x = TensorType(config.floatX, shape=(None, 1, 3))()
```

Comment:
```suggestion
        x = pt.tensor(shape=(None, 1, 3))
```
Why is TensorType being used directly here?